mysite/account/tests.py

```diff
@@ -287,26 +303,26 @@ def test_edit_email_address(self):
         assert new_email not in paulproteus_page
 
 
+# list of photos
 photos = [os.path.join(os.path.dirname(__file__),
           '..', '..', 'sample-photo.' + ext)
           for ext in ('png', 'jpg')]
 
 
 def photo(f):
+    """ Helper function for photo file """
     filename = os.path.join(os.path.dirname(__file__), '..', f)
     assert os.path.exists(filename)
     return filename
 
 
-@skipIf(not mysite.base.depends.Image, ("Skipping photo-related tests "
-                                        "because PIL is missing. Look in "
-                                        "ADVANCED_INSTALLATION.mkd for "
-                                        "information."))
```

Comment:
My only concern with this is that this might break the tests on a system where PIL(low) isn't installed.